src/psake.psd1

```diff
@@ -1,6 +1,6 @@
 @{
     RootModule        = 'psake.psm1'
-    ModuleVersion     = '4.8.1'
+    ModuleVersion     = '4.8.2'

```

Comment:
No need for the version bump. `4.8.1` hasn't been released yet. This PR would be added to it.